data/schema-config.yaml

```diff
@@ -314,6 +314,7 @@ regulatory region:
     start: int
     end: int
     biochemical_activity: str
+    biochemical_activity_description: str
```

Comment:
Since we add one more property **biochemical_activity_description**, should we also need to add it in filter_by and return in accessible_via?

Reply:
just added as a return. This is a text field, it would be a good fit for a `filter_by` exact string match.